google_auth_oauthlib/tool/__main__.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Command line tool for fetching credentials using 3LO OAuth2 flow.
```

Comment:
Recommendation: I've very much been concerned with preciseness of language in this library, maybe:

```
Command-line tool for obtaining authorization and credentials from a user.

This tool uses the OAuth 2.0 Authorization Code grant as described in `section 1.3.1 of RFC 6749`_
and implemeted by :class:`google_auth_oauthlib.flow.Flow`.

..  _section 1.3.1 of RFC 6749: https://tools.ietf.org/html/rfc6749#section-1.3.1
```

I'd recommend applying the terminology to the help message as well.